lib/services/search.js

```diff
@@ -358,13 +374,80 @@ function setup(overrideClient) {
  * endpoint. User is responsible for sending
  * the proper payload
  *
- * @param {object} queryObj [description]
+ * @param {object} queryObj
  * @returns {Promise}
  */
 function RESTQuery(queryObj) {
   return client.search(queryObj);
 }
 
+/**
+ * Load the mappings from the mappings folder.
+ *
+ * The name of the file becomes the name of the index (convention over configuration).
+ *
+ * @param {string} dirPath
+ * @returns {object}
+ */
+function retrieveMappingData(dirPath) {
+  var list = files.getFiles(dirPath),
+    mappings = {};
+
+  _.each(list, function (filename) {
+    const mappingName = filename.split('.')[0],
+      mapping = files.getYaml(path.join(dirPath, mappingName));
+
+    mappings[mappingName] = mapping;
+  });
+
+  return mappings;
+}
+
+// /**
+//  * Create indices that are required for Clay
+//  *
+//  * @return {Promise}
+//  */
+// function createInternalIndices() {
```

Comment:
I can remove this from this PR, but this function will be included in the next phase of changes whenever we actually instantiate the indices that are managed by Amphora.